tests/CATChem_config.yml

```
@@ -10,35 +10,29 @@ simulation:
   name: test
   start_date: 20240501 0000
   end_date: 20240501 0100
-  species_database_file: ./species_database.yml
-  species_metadata_output_file: ./species_metadata.yml
+  species_filename: CATChem_species.yml
```

```suggestion
  species_file: ./CATChem_species.yml
```
I would prefer this, since I think it makes it more clear that an abs/rel path name is expected, not just a file name?

---

Also in the config obj it seems to still be called `SpcDatabaseFile`? Maybe that should also be updated if accepting this suggestion.